packages/utils/src/treasury.ts

```diff
@@ -34,3 +36,7 @@ export const DAO_TREASURY_BASE = '0x9c9aA90363630d4ab1D9dbF416cc3BBC8d3Ed502'
 // Multisigs
 export const DAO_TREASURY_COSMOS = 'cosmos1qgmqsmytnwm6mhyxwjeur966lv9jacfexgfzxs'
 export const DAO_TREASURY_THORCHAIN = 'thor1xmaggkcln5m5fnha2780xrdrulmplvfrz6wj3l'
+export const DAO_TREASURY_SOLANA = 'C7RTJbss7R1r7j8NUNYbasUXfbPJR99PMhqznvCiU43N'
```

Comment:
This is the DAO address, to be confirmed with workstream leads if you want 

I added it because at some point Solana will be including fees but I can remove it if it's too early for you!